packages/side-nav/theme/lumo/vaadin-side-nav.js

```diff
@@ -3,6 +3,5 @@
  * Copyright (c) 2023 - 2024 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import './vaadin-side-nav-item.js';
```

Comment:
It looks like that would require you to import `@vaadin/side-nav/vaadin-side-nav-item` separately in addition to `@vaadin/side-nav`. For one it's a breaking change, plus it would be rather inconvenient as you can't do much with a side nav other than display side nav items in it. The dev page is also broken as a result of this.

Reply:
Also can't reproduce the warning in my test project if I restore this line 🤔

Reply:
Yeah, nevermind about the reproduction, looks like Vite handles it differently if the dependency comes from a file path. I can reproduce it fine if the package is in node_modules.

IMO it's a breaking change, you would have to modify your code to get it working again. Also, if you only import `@vaadin/side-nav` (which is what we use in docs), or `@vaadin/vaadin-side-nav.js` then you don't get the warning.

We also have several other packages that work similarly: details, accordion, radio-group, tabs. Would be a bit inconsistent if we change this.